torch/_dynamo/testing.py

```diff
@@ -93,6 +95,8 @@ def reduce_to_scalar_loss(out):
         return sum([reduce_to_scalar_loss(value) for value in out.values()]) / len(
             out.keys()
         )
+    elif isinstance(out, (int, bool)):
+        return int(out)
```

Comment:
This is pretty weird, why do we have an int loss. I think the algo may be misdescribed, this is minifier relevant right

Reply:
I think this was a hack to get some enablement fixed. I can pull it for now.